paystell-frontend/package.json

```
@@ -9,13 +9,16 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-slot": "^1.1.0",
     "@radix-ui/themes": "^3.1.5",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "lucide-react": "^0.456.0",
     "next": "14.2.17",
     "react": "^18",
     "react-dom": "^18",
+    "shadcn-ui": "^0.9.3",
```

Update 2 high severity vulnerabilities
```suggestion
    "shadcn-ui": "^0.2.3",
```